src/commands.js

```diff
@@ -29,25 +29,141 @@ commands.set('deposit', {
     try {
       // Submit the transfer transaction
       const dest = interaction.options.get('address', true).value;
+      if (!isValidAddress(dest)) throw new Error("Invalid Recipient");
       console.log(`Received deposit request for ${dest}`);
-      const api = await createApi('testnet');
       const mnemonic = process.env.SEED_PHRASE;
-      await transfer({
-        api, mnemonic, dest, amount: 1, onResult: (result) => {
-          if (result.status.isInBlock) {
-            const blockHash = result.status.asInBlock;
-            const link = 'https://testnet.avail.tools/#/explorer/query/' + blockHash;
+      const api = await initialize("wss://kate.avail.tools/ws")
```

Comment:
this needs to be changed, right? Can we take this too from some config to keep the URL customizable?